homeassistant/components/deconz/sensor.py

```diff
@@ -121,9 +123,43 @@ class DeconzSensorDescription(SensorEntityDescription, DeconzSensorDescriptionMi
         instance_check=AirQuality,
         name_suffix="PPB",
         old_unique_id_suffix="ppb",
+        device_class=None,
```

Comment:
```suggestion
```
This is None by default, no need to add it.

Reply:
Agree, this was left over. I had the device_class voc added but stumbled upon [Arch #858](https://github.com/home-assistant/architecture/discussions/858)
Therefore, I declared it None, but ofc could be completely omitted